indico/modules/rb/models/reservations.py

```diff
@@ -243,6 +243,11 @@ def __table_args__(cls):
         nullable=True,
         index=True
     )
+    notification_sent = db.Column(
```

Comment:
I guess this will change to `end_notification_sent`?

Reply:
yup, changed already